WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -903,6 +994,40 @@ + (BOOL)isNewEditorEnabled
 
 #pragma mark - Instance Methods
 
+- (void)keyboardWillHide:(NSNotification*)aNotification
+{
+    // If the format bar tooltip is still hanging around, let's git rid of it
+    if (self.formatBarToolTip) {
+        [self.formatBarToolTip cancelCurrentTooltip];
+        self.formatBarToolTip = nil;
+    }
+}
+
+- (void)keyboardWasShown:(NSNotification*)aNotification
+{
+    NSDictionary* info = [aNotification userInfo];
+    CGRect rawKeyboardRect = [[info objectForKey:UIKeyboardFrameEndUserInfoKey] CGRectValue];
+    CGRect kRect = [self.view.window convertRect:rawKeyboardRect toView:self.view.window.rootViewController.view];
+    
+    // We need to adjust the vertical space a little bit for the iPad and iPhone 6 in landscape
+    CGFloat verticalOffset = 0.0;
+    if (self.isLandscape && !WPDeviceIdentification.isiPhoneSixPlus) {
```

Comment:
No you did not. I originally had it running on the iPad and then decided not to display the tip there...I simply did not remove this code, but will.